tools/SourceKit/bindings/python/sourcekitd/capi.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, obj):
             self._obj = conf.lib.sourcekitd_request_dictionary_create(
                 POINTER(c_void_p)(), POINTER(c_void_p)(), 0)
             self._as_parameter_ = self._obj
-            for k,v in obj.iteritems():
+            for k,v in obj.items():
```

Comment:
Splitting hairs, but: you put a space in between `k,v` in your changes to `utils/GYBUnicodeDataUtils.py`. Why not place a space in between these two as well, to promote consistency?